lib/transaction.js

```diff
@@ -1,7 +1,7 @@
 const { apiRequest } = require('./apiRequest');
 
-async function getTransactionListByUser() {
-  const apiResponse = await apiRequest('GET', '/transaction/find-by-user', '?p=1', null);
+async function getTransactionListByUser(param='?p=1') {
+  const apiResponse = await apiRequest('GET', '/transaction/find-by-user', param, null);
```

Comment:
btw, what about making the third argument as array and do create the query parameters dynamically from inside the `apiRequest` using the following?

```
new URLSearchParams(param).toString()
```

I think this way the developer does not have to worry about appending the query string with `?` among other complexities. what do you think?

Reply:
Is it better to make it an array or key value pair object?

Reply:
key value pair, correct